app/build.gradle

```diff
@@ -614,6 +615,23 @@ task printVariants {
     }
 }
 
+task buildTranslationArray {
```

Comment:
Would you mind to move this to AC? :) We can have it as a Gradle plugin on the  `mozilla.components.support.locale`package, we have some examples of Gradle plugins in [buildSrc](https://github.com/psymoon/android-components/tree/08661a2af1d0afa09902daeec2423e40014006ce/buildSrc) and [here](https://github.com/psymoon/android-components/blob/c58d6a32d3c3815ced44fa65a811a42a51f0daee/build.gradle#L278). It will be pretty nice if we could generate an extension function on `LocaleManager` for people that use the plugging. Maybe [Kotlin Poet ](https://github.com/square/kotlinpoet#extension-functions)could be very helpful. What do you think?

Something like:

```kotlin
fun LocaleManager.getSupportedLocales(): List<Locale>{
    return listOf<Locale>(
            Locale("eu"),
            Locale("ca"),
            Locale("zh", "CN"),
            Locale("zh", "TW"),
            Locale("cs"),
            Locale("nl"),
            Locale("en"),
            Locale("fi"),
            Locale("fr"),
            Locale("de"),
            Locale("hu"),
            Locale("in"),
            Locale("it"),
            Locale("ja"),
            Locale("ko"),
            Locale("pl"),
            Locale("pa", "IN"),
            Locale("ru"),
            Locale("sk"),
            Locale("es"),
            Locale("es", "AR"),
            Locale("es", "ES"),
            Locale("sv", "SE"),
            Locale("ta"),
            Locale("tr")
    )
} 
```

Reply:
Hmm, turns out that it's not as straight-forward as I hoped at all. Looked into Kotlin Poet and the class generation is the doable part, but the following issues appeared:
- From what I see, declaring the task as a plugin in buildSrc makes it harder for me to share across projects
- Fenix needs to have it's own task, that simply calls the locale task in AC - currently I hardcode an absolute path to the locale task in AC, but I don't know exactly how to do that with the actual AC dependency
- AC task needs to access the res folder in fenix(and eventually other projects) in order to get the locales, and for that we need a path - currently I can't find how to send the path as a param (maybe how we do with ktlint?) - [here is their implementation](https://github.com/pinterest/ktlint/blob/master/ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt)

**TLDR**: I need to call an AC task from Fenix with params. If you have experience with this, I would appreciate the help! Otherwise, I would suggest going forward with this PR as is, since it will be working for Fenix and we're kinda on a short schedule for the planned release

Reply:
Let's me try to take a look at what you have done so far (Could you share your repo?). If it gets too complicated we can proceed as it is and improve it in a follow issue.

Reply:
Please, proceed with what you have so far here. I will try to find an alternative on AC.